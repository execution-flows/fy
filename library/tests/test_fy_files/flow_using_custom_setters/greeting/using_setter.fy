property greeting using setter:
    def -> str:
        return self.__greeting

    # Hello custom setter
    @_greeting.setter
    def _greeting(self, greeting: str) -> None:
        self.__greeting = greeting
