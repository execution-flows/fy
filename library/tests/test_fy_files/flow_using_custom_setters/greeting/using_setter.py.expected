class Greeting_UsingSetter_PropertyMixin:
    @property
    def _greeting(self) -> str:
        return self.__greeting

    # Hello custom setter
    @_greeting.setter
    def _greeting(self, greeting: str) -> None:
        self.__greeting = greeting
